### PR TITLE
WIP - Replace Otoken interest with amount

### PR DESCRIPTION
**Pull request by willeslau**
Remove supply borrow interest related functions. Use amount that sums principle + interest together.

Pending test cases

### CODE REVIEW

File: packages/ocean-lending/src/clients/oToken.ts [hunk 1]
```diff
@@ -145,6 +116,11 @@ class OToken extends Client {
     return BigInt(supply);
   }
 
+  public async totalBorrows(): Promise<BigInt> {
+    const supply = await this.contract.methods.totalBorrows().call();
+    return BigInt(supply);
```
Comment (khanhhaquang) [inline packages/ocean-lending/src/clients/oToken.ts]: I think just need to return as `string` type which is original type on chain

File: packages/ocean-lending/src/clients/oToken.ts [hunk 2]
```diff
@@ -183,56 +159,34 @@ class OToken extends Client {
...
-        this.borrowBalanceCurrent(account),
-        this.supplyInterest(account),
-        this.borrowInterest(account),
-        this.getCash()
-      ]);
+    const [supplyAPY, borrowAPY, supplyAmount, borrowAmount] = await Promise.all([
+      this.supplyAPY(blockTime),
+      this.borrowAPY(blockTime),
+      this.accountBorrowBalance(account),
+      this.accountBorrowBalance(account)
```
Comment (khanhhaquang) [inline packages/ocean-lending/src/clients/oToken.ts]: Duplicated here

**Review by khanhhaquang — CHANGES_REQUESTED** [review]
.

File: packages/ocean-lending/src/clients/oToken.ts (hunk 1 above)
Comment (willeslau) [reply packages/ocean-lending/src/clients/oToken.ts]: or return number

Comment (khanhhaquang) [reply packages/ocean-lending/src/clients/oToken.ts]: I think no need, just return it as pure as we can, 
the `Web3.utils.fromWei` is using param as string too

Comment (khanhhaquang) [reply packages/ocean-lending/src/clients/oToken.ts]: if return `number`, FE needs to convert again to string which is unnecessary

Comment (willeslau) [reply packages/ocean-lending/src/clients/oToken.ts]: because in the integration tests, we are doing compares.

Comment (willeslau) [reply packages/ocean-lending/src/clients/oToken.ts]: not sure if FE will need to perform math operations. I think as number it is easier for those operations.

Comment (khanhhaquang) [reply packages/ocean-lending/src/clients/oToken.ts]: Currently, FE doesn't have math operations, but if it had can easily convert to. Mostly using it for displaying

**Review by khanhhaquang — APPROVED** [review]

**Review by kun-dao — APPROVED** [review]

**Pull request merged**